test/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,25 @@
+package codeu.chat.util;
+import codeu.chat.util.Tokenizer;
+import org.junit.Test;
+import java.io.IOException;
+import static org.junit.Assert.*;
+public final class TokenizerTest {
```

Comment:
It is good you added few positive tests. I feel you can add tests for following cases which will add more coverage to your code:

1. empty string
2. string with just a beginning `"` but no closing `"`
3. string with just whitespaces
4. string with whitespaces in between